recipe/meta.yaml

```diff
@@ -12,16 +12,17 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -v --no-deps --no-build-isolation
 
 requirements:
   host:
     - pip
     - python
+    - setuptools
+    - wheel
     - setuptools_scm
   run:
-    - pytest >=3.6.0
+    - pytest
```

Comment:
pytest >= 3.7.0 ?
https://github.com/RKrahl/pytest-dependency/blob/cab2f65ced816939a9041b9e67169073ef0ee412/setup.py#L95

Reply:
addressed, thank you
(to >=3.6.0 as https://github.com/RKrahl/pytest-dependency/blob/0.5.1/setup.py#L59)